src/fsharp/FSharp.DependencyManager/FSharp.DependencyManager.fs

```diff
@@ -152,20 +157,28 @@ type [<DependencyManagerAttribute>] FSharpDependencyManager (outputDir:string op
                 if not (generatedScripts.ContainsKey(body.GetHashCode().ToString())) then
                     emitFile path  body
 
-            let fsProjectPath = Path.Combine(scriptsPath, "Project.fsproj")
+            let projectPath = Path.Combine(scriptsPath, "Project.fsproj")
```

Comment:
@brettfo, I did have it so it selected .csproj, instead of .fsproj, but I couldn't figure out a decent reason why.  For sure, the compilers are not touched by the targets we use, and so I switched it back.